doc/user_guide/user_guide.md

```diff
@@ -116,6 +116,26 @@ The option `--use-ssl-cert-validation`is the default, you can disable it with `-
 Use caution if you want to turn certificate validation off as it potentially lowers the security of your 
 Database connection.
 
+By default, the above command will upload and activate the language container at the System level.
+The latter requires you to have the System Privileges, as it will attempt to change DB system settings.
+If such privileges cannot be granted the activation can be skipped by using the `--no-alter-system` option.
+The command will then print two possible language activation SQL queries, which look like the following:
+```sql
+ALTER SESSION SET SCRIPT_LANGUAGES=...
+ALTER SYSTEM SET SCRIPT_LANGUAGES=...
+```
+These quires represent two alternative ways of activating a language container. The first one activates the
+container at the Session level. It doesn't require System Privileges. However, it must be run every time a
```

Comment:
we probably should also link to https://docs.exasol.com/db/latest/sql/alter_session.htm